### DISCUSSION TITLE
183

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#154] [General]
# 潮流周刊第183期 - 飞翔的鸟

封面图拍摄来源上周拍摄于西湖边玩轮滑的大爷，很 E 人，带着一个 BGM 播放着泰坦尼克号的主题曲，很开心的滑着，正如唐朝乐队 

https://weekly.tw93.fun/posts/183-%E9%A3%9E%E7%BF%94%E7%9A%84%E9%B8%9F

**Comment by Orange-Yellow** [comment]
置身事内这本书解读的太正确

**Comment by guguji5** [comment]
读后感很棒，我转发给身边好多朋友

**Reply by XiaoLiz** [reply]
这本书必须读

**Comment by klaybloom** [comment]
看到了不止一个人推荐这本书，写的很好
